include/IMeshBuffer.h

```diff
@@ -134,11 +136,20 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
         {0xdeadbeefu,8,16,24,32}
     };
 
+	//! Checks whether given combination of attribute's type and component count is valid
+	/**@param type Type of an atrribute.
+	@param components Component count of an atrribute.
+	@returns true if such combination is valid or false otherwise.
+	*/
```

Comment:
In OpenGL some formats are special like RGB30_A2 or RGB9_E5 which force the number of components and give no flexibility in the number of channels